types/express-session/index.d.ts

```diff
@@ -58,7 +59,17 @@ declare namespace session {
     secret: string | string[];
     name?: string;
     store?: Store | MemoryStore;
-    cookie?: express.CookieOptions;
+    cookie?: {
```

Comment:
@fiznool @sheetalkamat can we move this type to an own interface? Because I had a method before which was returning a shared config (for our microservices) the type `express.CookieOptions`.

Now after this PR I have to create manually an interface to keep the code. These types should provide all necessary types (including the cookie structure).

Reply:
Please submit the PR with the changes for these changes.

Reply:
@BorntraegerMarc you should be able to access this using TypeScript's [indexed access types](https://www.typescriptlang.org/docs/handbook/advanced-types.html#index-types):

``` ts
import { SessionOptions } from 'express-session';
type CookieOptions = SessionOptions['cookie'];
```

Reply:
I didn't know one can do this. Thanks a lot @fiznool !